core/metrics.py

```diff
@@ -1,19 +1,31 @@
+import sys
 from datetime import datetime
-from utils.web3_utils import init_skale_from_config
+
+from web3 import Web3
+
 from utils.filter import SkaleFilter
-import sys
 
 BLOCK_CHUNK_SIZE = 1000
 
 
-def get_nodes_for_validator(val_id):
-    skale = init_skale_from_config()
+def check_if_node_is_registered(skale, node_id):
+    if node_id in skale.nodes_data.get_active_node_ids():
```

Comment:
could be simplified to `return node_id in skale.nodes_data.get_active_node_ids()`

Reply:
done